packages/next/src/client/lib/is-chrome.ts

```diff
@@ -0,0 +1,17 @@
+export function isChromeDesktop() {
```

Comment:
This is not accurate. User agent parsing to detect features is generally discouraged. E.g. a masked agent will produce a false positive.

Reply:
I changed to isChrome with a loose checking, as it also doesn't harm to display there since it's a dev-only feature